### DISCUSSION TITLE
Using the Console through Service Accounts

### DISCUSSION THREAD

**Discussion by giovanni-motterle-mia-care** [mia-platform/community#288] [Q&A] [answered]
### Which section?

Console

### Which is the version?

latest

### Description

Hi, as described in the [documentation](https://docs.mia-platform.eu/docs/development_suite/identity-and-access-management/manage-service-accounts#what-can-you-do-with-a-service-account) it should be possible to write services that through a service account interact with the console (like to make automatic deployments).
Is there a documentation of all the APIs that this service could use?

Thanks

**Comment by fredmaggiowski** [comment, marked as answer]
Hi @giovanni-motterle-mia-care as of now we have a very limited set of APIs that can be used, we are not publishing them yet due to the fact that we want to consolidate them before and do not want to impact with breaking changes; the current set of API can be found in the [Console API Portal](https://console.cloud.mia-platform.eu/documentations/api-portal/#/api); please note that all APIs requiring for an API Key are not yet published and may be subject to breaking changes; if you need them you could still use them, you just have to ask us for an API Key

**Reply by giovanni-motterle-mia-care** [reply]
is there a roadmap of APIs releases? I'm particulary interested at interactions with the CRUD section of the console

**Reply by fredmaggiowski** [reply]
We have a roadmap but no specific dates, if you need specific api we may prioritize them :)

>I'm particulary interested at interactions with the CRUD section of the console

you mean things like "create a new collection from API"? This API does not exist and I don't thing will exist in the future; adding a new crud is part of the "save configuration" API; so this API should the one used to create a new crud, does it make sense to you?

**Reply by giovanni-motterle-mia-care** [reply]
is  "save configuration" API present in the API Portal you shared?

**Reply by fredmaggiowski** [reply]
not yet 😅

**Reply by giovanni-motterle-mia-care** [reply]
ok. I think it could be ok for what i have to, it would be nice if you can give priority to that.
Thanks

**Reply by fredmaggiowski** [reply]
@albertotessarotto @davidebianchi FYI
